db/models.py

```diff
@@ -1 +1,28 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
```

Comment:
The `bonus` field in the `Skill` model is currently defined as a `CharField`. It should be an `IntegerField` to store numeric values. Please change this field to `IntegerField` and remove the `max_length` parameter.